recipes/newrelic/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2.68.0.50" %}
+
+package:
+  name: newrelic
+  version: {{ version }}
+
+source:
+  fn: newrelic-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/n/newrelic/newrelic-{{ version }}.tar.gz
+  sha256: 3ac02183910fe41ab75485d05474164890b993d082dca5847b0e6d24cf166f89
+
+build:
+  number: 0
+  script:
```

Comment:
Yeah. 😞 Once we get this PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/14 ) in it shouldn't be. FWIW it seems it is now getting some love. So hopefully soon.